Window title

David-Mayes opened the pull request:
Resolves the issue.

Please let me know if the are any problems or slight improvements.

David-Mayes committed:
resolves #10: Window title should show current week

Added a get_current_week function to week-view
the function is used in main to create a title for the window.

David-Mayes committed:
fixed a few errors in my earlier commit

ohwgiles commented on main.c:
```diff
@@ -198,8 +198,14 @@ int main(int argc, char** argv)
+	int week = get_current_week(FOCAL_WEEK_VIEW(fm.weekView));
+	int length = snprintf(NULL, 0, "%d", week);
+	char week_num[5 + length + 1];
+	snprintf(week_num, 5 + length + 1, "Week %d", week);
```
Instead of two calls to printf, you could just `g_assert(week <= 53)` and then use a char array of fixed length `8`

ohwgiles replied:
Actually even asserting would be unnecessary since snprintf protects against the overflow and an out-of-range number is a programming error elsewhere

ohwgiles commented on week-view.h:
```diff
@@ -26,5 +26,6 @@ GtkWidget* week_view_new(void);
 void week_view_add_event(WeekView* cal, icalcomponent* vevent);
 void week_view_remove_event(WeekView* cal, icalcomponent* vevent);
 void week_view_add_calendar(WeekView* widget, Calendar* cal);
+int get_current_week(WeekView* wv);
```
Could this please be `week_view_get_current_week` or similar? Just to consistently follow glib/gtk naming conventions

ohwgiles reviewed (commented):
I have a couple of tiny suggestions but basically this is great, thanks!

David-Mayes committed:
improves #10: Window title should show current week

Removed unnecessary printf call.
Renamed function get_current_week to week_view_get_current_week to fit guidelines

ohwgiles commented:
Merged, thanks :)
